example/config.yml

```diff
@@ -213,3 +213,35 @@ keymap:
       Super-7: C-7
       Super-8: C-8
       Super-9: C-9
+
+# Absolute config section for tablet devices or touchscreens.
+# Missing fields default to 0,
+# so you don't have to set them if the desired value is 0
+absolute:
```

Comment:
I want this example to be a separate file because keyboards are still the primary use case of xremap. You may put something like `tablet.yml`.

Also, please consider updating `src/config/tests.rs`.